RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -4,32 +4,40 @@
 /** predicts phi range at a given radius r */
 
 #include "RecoTracker/TkMSParametrization/interface/PixelRecoRange.h"
-#include "RecoTracker/TkTrackingRegions/interface/TkTrackingRegionsMargin.h"
 
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include <cassert>
+
 class dso_internal OuterHitPhiPrediction {
 public:
 
-  typedef PixelRecoRange<float> Range;
-  typedef TkTrackingRegionsMargin<float> Margin;
+  using Range= PixelRecoRange<float>;
 
   OuterHitPhiPrediction( 
       const Range & phiAtVertex, 
       const Range & curvature, 
-      float originRBound, 
-      const Margin & tolerance = Margin(0,0))
+      float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
-      theOriginRBound (originRBound), theTolerance(tolerance) { } 
+      theOriginRBound (originRBound) {
+       assert(theCurvature.max()>0);
```

Comment:
For example, using: CMSSW_8_1_X_2016-06-08-2300, add PR #14811,
and execute ```addOnTests.py -t hlt_data_GRun``` - this works.
Then edit the file ```RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h:23```
and put back the ```assert``` and run the above addOn again. Then there is a crash!